server/server_other.go

```diff
@@ -27,7 +27,7 @@ func (s *Session) Namespace() (error, error) {
 	v("CPUD:namespace is %q", s.binds)
 
 	// Connect to the socket, return the nonce.
-	a := net.JoinHostPort("127.0.0.1", s.port9p)
+	a := net.JoinHostPort("localhost", s.port9p)
```

Comment:
the reason I used the raw ip was because /etc/hosts is not guaranteed to be there, but, are you doing this for ip6 / ip4 reasons? If so, I think I can drop my paranoia :)

Reply:
yeah, ipv4 isn't always there.  and i think "localhost" is handled by the go runtime magically, without involving /etc/hosts.

Reply:
i think